core/src/main/java/io/questdb/cairo/wal/seq/MetadataServiceStub.java

```diff
@@ -77,6 +78,10 @@ default boolean removePartition(long partitionTimestamp) {
         throw CairoException.critical(0).put("remove partition does not update sequencer metadata");
     }
 
+    @Override
+    default void setDeduplicationStatus(boolean status, LongList columnsIndexes, int startIndex, long count) {
```

Comment:
int columnLo, int columnHi will be more appropriate

Reply:
refactored signature, no lo, hi in 

https://github.com/questdb/questdb/pull/3524/commits/8699ee84624f26e3c01491745fc4ef3fd3e90633